staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer.go

```diff
@@ -133,6 +133,11 @@ func (g *Cloud) EnsureLoadBalancer(ctx context.Context, clusterName string, svc
 	if err != nil {
 		return nil, err
 	}
+	if err := checkMixedProtocol(svc.Spec.Ports); err != nil {
```

Comment:
the feature is GA now... why is this disabling support in current versions?

Reply:
it will be not supported in this controller, most probably not in the cloud-provider-gcp controller and in the long term in the ingress-gce controller